src/bin/trace-verifier/commands/run_rpc.rs

```diff
@@ -90,15 +91,15 @@ impl RunRpcCommand {
                             )
                             .await?;
 
-                        info!(
+                        dev_info!(
                             "worker#{idx}: load trace for block #{block_number}({})",
                             l2_trace.header.hash.unwrap()
                         );
 
                         let success = tokio::task::spawn_blocking(move || {
                             utils::verify(l2_trace, &fork_config, disable_checks, is_log_error)
                         })
-                        .await?;
+                        .await??;
```

Comment:
once you change `verify` to be `Result<(), VerificationError>` then you can use `is_ok()` here, instead of `if !success`